docs/modules/comments.md

```diff
@@ -0,0 +1,494 @@
+| [`CommentsPaginateRenderless`](#commentspaginaterenderless)         | A container used to render comments in a paginated view.                         |
+| [`CommentsTreeRenderless`](#resolvernames)                          | A container used to render comments in a hierarchical view.                      |
+
+### `CommentsNotificationRenderless`
+
+A component that displays a notification about new comments.
+
+**Usage**
+
+```js
+import React from 'react'
+import { CommentsNotificationRenderless } from '@resolve-js/module-comments'
+
+export const CommentsNotification = (props) => (
+  <CommentsNotificationRenderless
+    treeId="treeId"
+    parentCommentId="parentCommentId"
+    authorId="authorId"
+    checkInterval={10000} // default = 30 * 1000
+    readModelName="customReadModelName" // default = 'readModelName'
+    resolverName="customResolverName" // default = 'foreignCommentsCount'
+    {...props}
+  >
+    {({ count, onClick }) => {
+      // Replace with your code.
+      if (count === 0) return null
+      return (
+        <div onClick={onClick}>
+          Comments have been updated, refresh the page to see the new comments.
+        </div>
+      )
+    }}
+  </CommentsNotificationRenderless>
+)
+```
+
+The `CommentsNotificationRenderless` component takes the following props:
+
+| Prop Name         | Description                                                          |
+| ----------------- | -------------------------------------------------------------------- |
+| `checkInterval`   | The time interval in milliseconds between checks for updates.        |
+| `readModelName`   | The name of the comments read model.                                 |
+| `resolverName`    | The name of the comments read model's resolver.                      |
+| `onClick`         | A callback function that handles a user's click on the notification. |
+| `treeId`          | The unique identifier of the current comment tree.                   |
+| `parentCommentId` | The unique identifier of the parent comment.                         |
+| `authorId`        | The comment author's unique identifier.                              |
+
+This component passes all its props to its direct child.
+
+The child component also receives the following additional props:
+
+| Prop Name | Description                                                                                                               |
+| --------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `onClick` | Resets the notification and calls the `CommentsNotificationRenderless` component's `onClick` callback if it is specified. |
+| `count`   | The number of available new comments.                                                                                     |
```

Comment:
What do you mean by "available"? Is this word necessary?

Reply:
I think it does not add much. Changed to "The number of new comments."

Reply:
Thanks.